compiler/luci/import/src/Nodes/CircleBCQFullyConnected.cpp

```diff
@@ -41,6 +43,7 @@ CircleNode *CircleBCQFullyConnectedGraphBuilder::build_node(const circle::Operat
   node->weights_scales(inputs[1]);
   node->weights_binary(inputs[2]);
   node->bias(inputs[3]);
+  node->weights_clusters(inputs[4]);
```

Comment:
maybe add `if (args.op.inputs.size() >= 5)` or something?

Reply:
Hmm... if I understand correctly, that checking is already done at line 30 :)
```cpp
  if (args.op.inputs.size() != 4 && args.op.inputs.size() != 5)
    return false;
```

Reply:
if `args.op.inputs.size() == 4 ` then `inputs[4]` will make out of bounds error

Reply:
@seanshpark Thanks for the sharp review :) I will fix related issues first and then ping again..!